examples/scatter-matrix-app.py

```diff
@@ -0,0 +1,42 @@
+from dash import Dash, dcc, html, Input, Output
+import pandas as pd
+import plotly.express as px
+
+filepath = "https://raw.githubusercontent.com/plotly/datasets/master/diabetes.csv"
+df = pd.read_csv(filepath)
+# fig = px.scatter_matrix(df)
+# fig.show()
+dimensions_list = df.columns.values
+
+app = Dash(__name__)
+
+app.layout = html.Div([
+    html.H1("Scatterplot Matrix of the Correlates of Diabetes",
+            style={'textAlign': 'center'}),
+    html.Div(
+        [html.H3('Choose your dimensions:',
+                 style={"paddingLight": "60px"}),
+         dcc.Dropdown(
+            id='scatter-matrix-app-x-dimension-dropdown',
+            options=dimensions_list,
+            value=["Outcome", "Glucose", "BMI"],
+            multi=True
+        )], style={"display": "inline-block", "verticalAlign": "top", "width": "30%", "margin-left": "5em"}),
```

Comment:
The `"display": "inline-block"` and  `"verticalAlign": "top"` don't have any effect and could be removed.

`"width": "30%"` looks nice on a large screen, but it's pretty squishy on a small one.  What do you think about using 50%?

`"margin-left"` should be `"marginLeft"`

Reply:
It's weird because I specifically liked that "margin-left" aligned the left side of the dropdown with the graph. But then maybe it's just the way it looks on my screen and it's different on yours.  See the screenshot below. 
![image](https://user-images.githubusercontent.com/31133836/177755536-5f8eaa83-44ea-49b0-a6c8-8fb9f8e8d534.png). Anyhow, I have replaced "margin-left" by "marginLeft" as requested.

Reply:
done!